test/parallel/test-tls-handshake-error.js

```diff
@@ -19,7 +19,7 @@ const server = tls.createServer({
   assert.throws(() => {
     tls.connect({
       port: this.address().port,
-      ciphers: 'RC4'
+      ciphers: 'missing'
```

Comment:
nit: maybe 'no-such-cipher' would be clearer?

Reply:
Sounds good, I'll update shortly. Thanks